src/ngAnimate/animateCss.js

```diff
@@ -749,11 +747,6 @@ var $AnimateCssProvider = ['$animateProvider', function($animateProvider) {
           options.onDone();
         }
 
-        // Remove the transitionend / animationend listener(s)
-        if (events) {
-          element.off(events.join(' '), onAnimationProgress);
```

Comment:
Nt sure if it is related to the "breakage", but this line can cause trouble if `events` is an empty array. This is because of how `jqLiteOn` and `jqLiteOff` work (I didn't look into jQuery's implementation).

With an empty `events` array, we will be calling `element.on('', onAnimationProgress)` and `element.off('', onAnimationProgress)`. Due to how `.on()` and `.off()` are implemented, these two calls will result in:

1. `.on(...)`: Register a listener for the `''` (empty string) event (won't ever fire - but that's fine).
2. `.off(...)`: Deregister **all** listeners, because: `.off('', ...)` <==> `.off()`.
    `jqLiteOff` only check if the `event` is truthy (and `''` isn't) and if not, it thinks it is a no-argument call and deregisters all listeners.